src/service/icon/icon_generator.ts

```diff
@@ -7,7 +6,7 @@ import constants from 'src/shared/const/iconGen';
 
 @Injectable()
 export class ImageGen {
-  public static generateImage(email: string): Promise<Buffer> {
+  public generateImage(email: string): Promise<Buffer> {
```

Comment:
Why `Promise<Buffer>`?

Reply:
@tedi4t Because the method return getBufferAsync will return this type
```javascript
public generateImage(email: string): Promise<Buffer> {
  ...
  return img.getBufferAsync(Jimp.MIME_JPEG);
}
```